examples/introExample/src/main/java/io/realm/examples/intro/IntroExampleActivity.java

```diff
@@ -77,54 +69,74 @@ protected void onDestroy() {
         realm.close(); // Remember to close Realm when done.
     }
 
-    private void showStatus(String txt) {
-        Log.i(TAG, txt);
-        TextView tv = new TextView(this);
-        tv.setText(txt);
-        rootLayout.addView(tv);
+    private void showStatus(String text) {
+        Log.i(TAG, text);
+        TextView textView = new TextView(this);
+        textView.setText(text);
+        rootLayout.addView(textView);
     }
 
     private void basicCRUD(Realm realm) {
         showStatus("Perform basic Create/Read/Update/Delete (CRUD) operations...");
 
-        // All writes must be wrapped in a transaction to facilitate safe multi threading
-        realm.executeTransaction(new Realm.Transaction() {
-            @Override
-            public void execute(Realm realm) {
-                // Add a person
-                Person person = realm.createObject(Person.class);
-                person.setId(1);
-                person.setName("Young Person");
-                person.setAge(14);
+        // Results obtained from a Realm are live, and can be observed on looper threads (like the UI thread).
+        // Note that if you want to observe the RealmResults for a long time, then it should be a field reference.
+        // Otherwise, the RealmResults can no longer be notified if the GC has cleared the reference to it.
+        RealmResults<Person> persons = realm.where(Person.class).findAll();
 
+        // The change listener will be notified when the Realm is written to from any threads.
+        persons.addChangeListener((people, changeSet) -> {
+            if(changeSet == null) {
```

Comment:
With 5.0, this can never be `null` anymore. Instead check for `changeSet.getState() == State.INITITAL`

Reply:
I was using `master` branch so this didn't have `changeSet.getState()` yet ._.